src/wildcard.ts

```diff
@@ -279,6 +279,9 @@ export const DEFAULT_ENUM_INDEX: EnumIndex = {
 export function getDefaultEnumValues(prop: Property, schema: Schema, opt: QueryConfig): any[] {
   if (prop === 'field' || (isEncodingNestedProp(prop) && prop.parent === 'sort' && prop.child === 'field')) {
     // For field, by default enumerate all fields
+    if ((isEncodingNestedProp(prop) && prop.parent === 'sort' && prop.child === 'field') && opt.autoAddCountOnSort){
```

Comment:
The insertion here is a bit weird because it makes the following comment outdated:
```
For field, by default enumerate all fields
```

The name `autoAddCountOnSort` is a bit misleading since you are not really adding count here, you are adding `'*'`.   

I'd say remove the misleading `autoAddCountOnSort` config and just keep this logic here, but add proper comment describing why you need to add '*' to `sort.field` (to support count).